sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_utils.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from typing import cast
+from xml.etree.ElementTree import ElementTree
+
+
+import urllib.parse as urlparse
+
+from azure.servicebus.management import _constants as constants
+from ...management._handle_response_error import _handle_response_error
+
+
+async def extract_data_template(feed_class, convert, feed_element):
+    deserialized = feed_class.deserialize(feed_element)
+    list_of_qd = [convert(x) if convert else x for x in deserialized.entry]
+    next_link = None
+    if deserialized.link and len(deserialized.link) == 2:
+        next_link = deserialized.link[1].href
+    return next_link, iter(list_of_qd)
+
+
+async def get_next_template(list_func, *args, **kwargs):
+    if args:
+        next_link = args[0]
+    else:
+        next_link = kwargs.pop("next_link")
+
+    start_index = kwargs.pop("start_index", 0)
+    max_page_size = kwargs.pop("max_page_size", 100)
```

Comment:
as I touched on in another comment, since these are aio, why can't we use keyword-only args with sane defaults to make this all visible and more explicit?

Reply:
Changed to the following for async version.
```python
async def get_next_template(list_func, *args, start_index=0, max_page_size=100, **kwargs):
```

Reply:
For future reference, you don't need to write *args, just writing * is sufficient and communicates the intent that you won't be utilizing that var.  It's like "_" in other situations.  (This isn't major enough that I want to block you on it, just a thought for the future)